relocation.bs

```diff
@@ -329,7 +330,6 @@ A `reloc` statement is ill-formed if any of the following conditions is met:
     value) ;
 - the source object is not ref-qualified and does not have an accessible relocation,
     move, or copy constructor ;
-- the source object is a structured binding ;
```

Comment:
Again, 'structured bindings' are aliases and they cannot be relocated.
With the way I defined structured relocation, the identifiers are not structured bindings but individual objects.
I get this may be confusing, I'll give it some thoughts.